jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -34,7 +34,7 @@ describe('EntityDetails', function() {
   it('can be rendered', function() {
```

Comment:
This suite is missing the tests for the multiple entity details rendering outputs.